### PR TITLE
[cherry-pick] Review installation (rhcarvalho)

### PR DESCRIPTION
**Pull request by danmacpherson**
#3124 

Reviewing and cherrypicking commits from @rhcarvalho 

Some commits from #3124 no longer apply.

### CODE REVIEW

**Comment by danmacpherson** [comment]
@ahardin-rh @vikram-redhat -- Made a new MR and cherry picked from the original MR from @rhcarvalho. Some commits no longer. Have also squashed into one commit.

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -113,19 +113,19 @@ production environments, the following recommendations apply:
...
-should have 1 CPU core and 1.5 GB of memory, on top of the defaults in the table
-above, for each 1000 pods. Therefore, the recommended size of master host in an
-{product-title} cluster of 2000 pods would be 2 CPU cores and 3 GB of RAM, in
-addition to the minimum requirements for a master host of 2 CPU cores and 16 GB
-of RAM.
+should have, in addition to the minimum requirements in the table above, 1 CPU
+core and 1.5 GB of memory for each 1000 pods. Therefore, the recommended size of
+a master host in an {product-title} cluster of 2000 pods would be the minimum
+requirements of 2 CPU cores and 16 GB of RAM, plus 2 CPU cores and 3 GB of RAM,
+totaling 4 CPU cores and 19 GB of RAM.
```
Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: @jeremyeder Can you please confirm that these numbers are correct? Thanks!

Comment (danmacpherson) [inline install_config/install/prerequisites.adoc]: @ahardin-rh just a note: the numbers should essentially be the same as before. The wording has changed to clarify the total requirements.

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: @danmacpherson Okay, thanks!

**Comment by ahardin-rh** [comment]
@danmacpherson I think this is ready to move forward. We just need to set labels and milestone. :fireworks:

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**